ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp

```diff
@@ -168,6 +240,7 @@ uint32_t RSkTextLayoutManager::buildParagraph (AttributedString attributedString
             style.setBackgroundColor(convertTextColor(fragment.textAttributes.backgroundColor ?
                                                       fragment.textAttributes.backgroundColor :
                                                       TextAttributes::defaultTextAttributes().backgroundColor));
+            style.setDecorationColor(convertTextPropColor(fontDecorationColor));      
```

Comment:
convertTextColor function can be reused here instead of convertTextPropColor

Reply:
Updated